### DISCUSSION TITLE
[Bug] Found 0 files in /MyTTSDataset/wav which has 300 of my data

### DISCUSSION THREAD

**Discussion by elhamadjidi** [coqui-ai/TTS#1612] [General Q&A]
### Describe the bug

Fine tuning gives me an error

I made the dataset with the same format mentioned here https://tts.readthedocs.io/en/stable/formatting_your_dataset.html#using-your-dataset-in-tts  but when I run below code, it gives me an error saying "Found 0 files in /MyTTSDataset/wav which has 300 of my data:

```
from TTS.tts.configs.shared_configs import BaseDatasetConfig
from TTS.tts.datasets import load_tts_samples


# dataset config for one of the pre-defined datasets
dataset_config = BaseDatasetConfig(
    name="vctk", meta_file_train="", language="en-us", path="dataset-path")
)

# load training samples
train_samples, eval_samples = load_tts_samples(dataset_config, eval_split=True)
```
Is this a bug or my problem?

### To Reproduce

from TTS.tts.configs.shared_configs import BaseDatasetConfig
from TTS.tts.datasets import load_tts_samples

# dataset config for one of the pre-defined datasets
dataset_config = BaseDatasetConfig(name="vctk", meta_file_train="metadata.txt", language="en-us", path="/MyTTSDataset/wav")

# load training samples
train_samples, eval_samples = load_tts_samples(dataset_config, eval_split=False)

### Expected behavior

making the formatter and load train-test

### Environment

```shell
python 3.9
```

**Comment by thorstenMueller** [comment]
Hi,
just a few guesses:

> meta_file_train=""

You have to pass a metadata file matching the files in your dataset folder.

> path="dataset-path"

Is "dataset-path" a variable or a static folder (because of the quotes)?

Is path correct (case sensitive)?

**Comment by elhamadjidi** [comment]
Hi 

> Hi, just a few guesses:
> 
> > meta_file_train=""
> 
> You have to pass a metadata file matching the files in your dataset folder.
> 
> > path="dataset-path"
> 
> Is "dataset-path" a variable or a static folder (because of the quotes)?
> 
> Is path correct (case sensitive)?

Hi this is the code `dataset_config = BaseDatasetConfig(name="vctk", meta_file_train="metadata.txt", language="en-us", path="/MyTTSDataset/wav")` as you can see I put the metadata.txt for meta_file_train and for path I used the absolute path, I just didn't share the user/.... part.
And the path is correct because I copied and pasted it.

**Comment by elhamadjidi** [comment]
the MyTTSDataset has two items, one is a metadata.txt file and one is a wav folder which consists of 300 wav files.

**Comment by WeberJulian** [comment]
Hey, use vctk_old as a formatter, the current format uses flac files, not wavs https://github.com/coqui-ai/TTS/blob/f237e4ccd9f2fd2b0cb5e136dfe0e20cc32bf898/TTS/tts/datasets/formatters.py#L337

**Comment by Ashoka74** [comment]
Same issue here, did you manage to solve it ? Changing the formatter didn't solve the issue for me.

**Reply by elhamadjidi** [reply]
> Same issue here, did you manage to solve it ? Changing the formatter didn't solve the issue for me.

No, I couldn't solve this problem, now I am trying another library.

**Reply by WeberJulian** [reply]
Hey @Ashoka74, make sure that your data is formatted the same way as dataset the formatter you use is made for.
If your data is formatted for VCTK and you use wav instead of flac, use `vctk_old` instead.

**Comment by WeberJulian** [comment]
Hey @elhamadjidi, I think you just were using the wrong formater for your data.
If you want to use the [VCTK formatter](https://github.com/coqui-ai/TTS/blob/f70e82cd1953c2d1e03c54add7d5dfbdecc8d0bb/TTS/tts/datasets/formatters.py#L307) you need to have data formated the same way as VCTK, so that way:

```
vctk_dataset
    txt
        speaker_1
            sentence_1.txt
            sentence_2.txt
        speaker_2
            sentence_1.txt
            sentence_2.txt
    wav48
        speaker_1
            sentence_1.wav
            sentence_2.wav
        speaker_2
            sentence_1.wav
            sentence_2.wav
```

Since your dataset is like this:
```
dataset
    metadate.txt
    wavs
        file1.wav
        file2.wav
```
You might want to use the [LJSpeech dataset formatter](https://github.com/coqui-ai/TTS/blob/f70e82cd1953c2d1e03c54add7d5dfbdecc8d0bb/TTS/tts/datasets/formatters.py#L111). Be mindfull of the separator.
